server/schema/submissions/createSubmission.js

```diff
@@ -0,0 +1,32 @@
+const schema = {
+  required: ['output'],
```

Comment:
Oh..totally forgot this as well. ouput should be required only for subjective, 
```suggestion
```

Reply:
being solved under new PR